src/CircuitBreaker.ts

```diff
@@ -49,6 +51,10 @@ export class CircuitBreaker extends EventEmitter {
         return this._aggregateState;
     }
 
+    get promErr(): boolean {
```

Comment:
should not be limited to prometheus : if there is an error, we should be able to handle it...

Reply:
https://github.com/scality/breakbeat/pull/11/files#diff-359e4042f12fa2b33c080ef07d2ff300cdd9137a3f55c0dc89b5d823527da29fR54-R57